Formula/vvdec.rb

```diff
@@ -0,0 +1,27 @@
+class Vvdec < Formula
+  desc "Fraunhofer Versatile Video Decoder"
+  homepage "https://github.com/fraunhoferhhi/vvdec"
+  url "https://github.com/fraunhoferhhi/vvdec/archive/refs/tags/v1.6.1.tar.gz"
+  sha256 "72de0cbcd24285e6f66209be9270f8f0c897d24e586b3876c6a7bb5691375c48"
+  license "BSD-3-Clause-Clear"
+  head "https://github.com/fraunhoferhhi/vvdec.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  resource("test_video") do
+    url "https://github.com/pthariensflame/homebrew-assorted/raw/master/test-fixtures/test.vvc"
```

Comment:
Could probably upload this to Internet Archive. Might be safer in case something happens to your account/repo?